### PR TITLE
fix: worflow run on renovate branches

### PR DESCRIPTION
**Pull request by frezbo**
Fix workflow run being triggered on renovate branches.

Also bump deps.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on:
       - self-hosted
       - generic
-    if: (!startsWith(github.head_ref, 'renovate/') || !startsWith(github.head_ref, 'dependabot/'))
+    if: ${{ (!startsWith(github.head_ref, 'renovate/') || !startsWith(github.head_ref, 'dependabot/')) }}
```
Comment (smira) [inline .github/workflows/ci.yaml:28]: should it be `&&` here?

Comment (frezbo) [reply .github/workflows/ci.yaml:28]: no, dependabot or renovate and it's a not

Comment (frezbo) [reply .github/workflows/ci.yaml:28]: yes, should be `&&`

Comment (frezbo) [reply .github/workflows/ci.yaml:28]: fixed

File: internal/output/ghworkflow/gh_workflow.go [hunk 2]
```diff
@@ -48,10 +48,10 @@ func NewOutput() *Output {
 			ciWorkflow: {
 				Name: "default",
 				// https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value
-				Concurrency: Concurrency{
-					Group:            "${{ github.head_ref || github.run_id }}",
-					CancelInProgress: true,
-				},
+				// Concurrency: Concurrency{
```
Comment (frezbo) [inline internal/output/ghworkflow/gh_workflow.go:51]: keeping commented since I found an edge case when the PR is initially opened it will get cancelled

**Review by smira — APPROVED** [review]

**Pull request merged**